### PR TITLE
Setting up authentication (sign-up and log-in) with devise

### PR DESCRIPTION
**Pull request by danmalmx**
## PT Link
https://www.pivotaltracker.com/story/show/161245978

## Included in PR
- Installed Devise gem
- Installed Devise views
- user can sign up to page
- user can also login to their account
- Added styling for sign-up and Log-in
- Added functionality to show/hide Sign-Up, Log-In and Log-Out buttons - depending on user authentication stage. 

## Screen Shot

screenshot 2018-10-25 at 09 27 16

screenshot 2018-10-25 at 09 27 22

### CODE REVIEW

File: app/views/devise/confirmations/new.html.erb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<h2>Resend confirmation instructions</h2>
```
Comment (Rinbo) [inline app/views/devise/confirmations/new.html.erb]: Do we need this form? Is it being used?

File: app/views/devise/mailer/confirmation_instructions.html.erb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+<p>Welcome <%= @email %>!</p>
```
Comment (Rinbo) [inline app/views/devise/mailer/confirmation_instructions.html.erb]: Are we using any of the devise mailer views?

File: app/views/devise/passwords/edit.html.erb [hunk 3]
```diff
@@ -0,0 +1,25 @@
+<h2>Change your password</h2>
```
Comment (Rinbo) [inline app/views/devise/passwords/edit.html.erb]: Are we gonna use this? Is there a link to it?

Comment (Noelryn) [reply app/views/devise/passwords/edit.html.erb]: There is no link for it,  but should we leave it for later use to changed a users password or is this out of scope

File: app/views/devise/confirmations/new.html.erb (hunk 1 above)
Comment (Noelryn) [reply app/views/devise/confirmations/new.html.erb]: I can remove this now

**Pull request merged**